### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Assessments results and recommendations_Status Messages: Narrator/NVDA is not announcing "Assessments in progress" loading status

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19885]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Github Tags:
#A11yMAS; #A11yTCS; #A11yWCAG2.1; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG4.1.3; #Status Messages; #NVDA; #Narrator;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Repro Steps:

1. Turn on Narrator/NVDA
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment" in step 1, select few databases and click on 'next' button
7. Navigate to "Assessment results and recommendations" and verify that screen reader is not announcing "Assessments in progress" loading status. 

### Actual Result:
Narrator/NVDA is not announcing "Assessments in progress" loading status in "Assessment results and recommendations" step

### Expected Result:
Narrator/NVDA should announce "Assessments in progress" loading status in "Assessment results and recommendations" step

### Attachment:
![NVDA_status message](https://user-images.githubusercontent.com/83570638/176439888-c6ad947f-f7c9-4416-9e45-d2e00ea7f3a1.png)
![Status messages_](https://user-images.githubusercontent.com/83570638/176439970-584bc988-0afe-46b9-a28d-ce405316116c.gif)

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in below Environment:
Issue is Fixed
Version: 1.41.1 (user setup)
![Screenshot (31)](https://user-images.githubusercontent.com/100189407/218056149-b3fcf4bf-c95b-42a7-86b8-a11310d69b02.png)
![Screenshot (30)](https://user-images.githubusercontent.com/100189407/218056160-2975a26c-33fb-4532-b4c7-21e39f2d5e74.png)
